src/Microsoft.Azure.WebJobs.Host/Blobs/Bindings/BlobAttributeBindingProvider.cs

```diff
@@ -60,7 +60,7 @@ internal class BlobAttributeBindingProvider : IBindingProvider
 
             string resolvedPath = Resolve(blobAttribute.BlobPath);
             IBindableBlobPath path = null;
-            IStorageAccount account = await _accountProvider.GetStorageAccountAsync(context.Parameter, context.CancellationToken, _nameResolver);
```

Comment:
are there still valid uses of the `GetStorageAccountAsync(parameterInfo ... )` call? might be able to remove it